r-package/grf/R/causal_forest.R

```diff
@@ -308,7 +310,9 @@ predict.causal_forest <- function(object, newdata = NULL,
                                   linear.correction.variables = NULL,
                                   ll.lambda = 0.1,
                                   ll.weight.penalty = FALSE,
-                                  num.threads = NULL, estimate.variance = FALSE, ...) {
+                                  num.threads = NULL,
+                                  estimate.variance = FALSE,
+                                  tune.parameters = FALSE, ...) {
```

Comment:
I'm wondering why we have this new parameter `tune.parameters`, whereas in `regression_predict` we don't? It would be good to make these two methods consistent. I also noticed that here `ll.lambda` defaults to `0.1`, whereas for `regression_predict` it defaults to `NULL`.